spec/models/course/forum_spec.rb

```diff
@@ -24,5 +24,41 @@
         end
       end
     end
+
+    describe '.with_topic_count' do
+      let(:forum) { create(:forum) }
+      let!(:forum_topics) { create_list(:forum_topic, 2, forum: forum) }
+
+      it 'shows the correct count' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.